Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'sinatra',         '~> 1.4.2'
 gem 'rake',            '~> 0.9.2.2'
 gem 'multi_json'
 
-gem 'sentry-raven',    github: 'getsentry/raven-ruby'
+gem 'sentry-raven',    git: 'https://github.com/getsentry/raven-ruby'
```

Comment:
My worry is that right now this PR is updating to a SHA that isn't officially released at all, and it'll be hard to tell if there are any known bugs in it at all, since there are several commits both before and after this one. There are also several breaking changes between the old version and the new version, are those checked to make sure we don't need to update things in listener to handle those?

I'm happy to take this over and check those things if you don't want to.